pallets/runtime/src/general_tm.rs

```diff
@@ -208,22 +249,118 @@ decl_module! {
+            ensure!(<Identity<T>>::is_identity_exists(&trusted_issuer), Error::<T>::DidNotExist);
+            Self::unsafe_modify_default_trusted_claim_issuer(ticker, trusted_issuer);
+            Ok(())
+        }
+
+        /// To add/remove the default trusted claim issuer for a given asset
+        /// Addition - When the given element is not exist
+        /// Removal - When the given element is already present
+        ///
+        /// # Arguments
+        /// * origin - Signer of the dispatchable. It should be the owner of the ticker.
+        /// * ticker - Symbol of the asset.
+        /// * trusted_issuers - Vector of IdentityId of the trusted claim issuers.
+        fn modify_default_trusted_claim_issuers_batch(origin, ticker: Ticker, trusted_issuers: Vec<IdentityId>) -> DispatchResult {
+            let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
+            let did = Context::current_identity_or::<Identity<T>>(&sender_key)?;
+            let sender = Signatory::AccountKey(sender_key);
+
+            ensure!(
+                <Identity<T>>::is_signer_authorized(did, &sender),
+                Error::<T>::SenderMustBeSigningKeyForDid
+            );
+            ensure!(trusted_issuers.len() >= 1, Error::<T>::InvalidLength);
+            ensure!(Self::is_owner(&ticker, did), Error::<T>::Unauthorized);
+            // ensure whether the trusted issuer's did is register did or not
+            for trusted_issuer in trusted_issuers.iter() {
+                ensure!(<Identity<T>>::is_identity_exists(trusted_issuer), Error::<T>::DidNotExist);
+            }
+            // iterate all the trusted issuer and modify the data of those.
+            for default_issuer in trusted_issuers.into_iter() {
+                Self::unsafe_modify_default_trusted_claim_issuer(ticker, default_issuer);
+            }
+            Ok(())
+        }
+
+        /// Change/Modify the existing asset rule of a given ticker
+        ///
+        /// # Arguments
+        /// * origin - Signer of the dispatchable. It should be the owner of the ticker.
+        /// * ticker - Symbol of the asset.
+        /// * asset_rule_id - Unique id of the asset rule.
+        /// * new_asset_rule - new asset rule.
+        fn change_asset_rule(origin, ticker: Ticker, asset_rule_id: u32, new_asset_rule: AssetRule) -> DispatchResult {
+            let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
+            let did = Context::current_identity_or::<Identity<T>>(&sender_key)?;
+            let sender = Signatory::AccountKey(sender_key);
+
+            ensure!(
+                <Identity<T>>::is_signer_authorized(did, &sender),
+                Error::<T>::SenderMustBeSigningKeyForDid
+            );
+            ensure!(Self::is_owner(&ticker, did), Error::<T>::Unauthorized);
+            ensure!(Self::asset_rules(ticker).latest_rule_id >= asset_rule_id, Error::<T>::InvalidRuleId);
```

Comment:
Since this is actually `next_rule_id`, the comparison should be `>` instead of `>=`.